src/Prism.Core/Mvvm/ErrorsContainer.cs

```diff
@@ -12,7 +12,14 @@ namespace Prism.Mvvm
     public class ErrorsContainer<T>
     {
         private static readonly T[] noErrors = new T[0];
+        /// <summary>
```

Comment:
put a space between the xml docs and the previous line of code

Reply:
Space added